xrdp/lang.c

```diff
@@ -297,3 +281,33 @@ get_keymaps(int keylayout, struct xrdp_keymap *keymap)
     g_free(filename);
     return 0;
 }
+
+/*****************************************************************************/
+int km_load_file(const char *filename, struct xrdp_keymap *keymap)
+{
+    int fd;
+
+    fd = g_file_open(filename);
+    LOG(LOG_LEVEL_INFO, "Loading keymap file %s", filename);
+
+    if (fd != -1)
```

Comment:
Need to log error if keymap couldn't be opened

Reply:
Indeed.